functions/home-tab.js

```diff
@@ -16,14 +16,43 @@ async function renderHomeTab(event) {
     }
   } = await client.users.info({ user: userId });
 
+  const allItems = ...await listCommandBlock({ userId, teamId });
+  const myItems = ...await getMylistBlocks({ userId });
+  const mySoldItems = ...await getMyListHistoryBlocks({ userId });
+
+  // build blocks based on returned values form queries
+  const blocks = [
+    headerBlock('Welcome :partying_face: \n Check out the items in the marketplace! :kite:'),
+    divider,
+  ];
+
+  if (allItems == []) {
```

Comment:
```suggestion
  if (allItems.length) {
```
it would return 0, and 0 is falsy

Reply:
this is NIT. doesn't matter that much